docs/source/upgrade-guide/index.md

```diff
@@ -30,6 +30,82 @@ the latest changes, and propose to you to merge the changes, so you can run it o
 
 ## Upgrading to Volto 16.x.x
 
+### `volto-slate` is now core
+
+From Volto 16.0.0-alpha.15 onwards, `volto-slate` is integrated into Volto core and enabled by default. This means that the default text block is now called `slate` and the former/deprecated text block `text` based on `draftJS` is now restricted (hidden) and enforced on Volto bootstrap. This is a major change and should be planned in advance before you install any version of the alphas after 16.0.0-alpha.15.
```

Comment:
In the future, please use one sentence per line, per Plone documentation guidelines. It makes reviews and diffs a little easier.

I would like to start using standard Sphinx admonitions regarding features going forward as well. The admonitions are `versionadded`, `versionchanged`, and `deprecated`. See them in action in a theme on which ours is based:

https://sphinx-themes.org/sample-sites/furo/kitchen-sink/admonitions/#versionadded

The phrase "is now restricted (hidden) and enforced on Volto bootstrap." seems contradictory to itself. How can it be both restricted and enforced (implies it is required)? As such, I revised that sentence to what I think you intend.

````suggestion
From Volto 16.0.0-alpha.15 onwards, `volto-slate` is integrated into Volto core and enabled as the default text block.
The previous text block `text` based on `draftJS` is now deprecated and is restricted (hidden) in Volto bootstrap.
This is a major change and should be planned in advance before you install 16.0.0-alpha.15 or later.

```{versionadded} 16.0.0-alpha.15
`volto-slate` added to Volto core as the default text block.
```

```{deprecated} 16.0.0-alpha.15
`text` text block based on `draftJS` is now deprecated and might be removed from core in a future major version.
```
````